src/preview/browser/statics/template.html

```diff
@@ -14,4 +14,8 @@
     {font_previews}
 </div>
 </body>
-</html>
\ No newline at end of file
+</html>
+
+<script>
```

Comment:
Please move this `script` element to the end of `body`, due to the [HTML standard](https://html.spec.whatwg.org/multipage/semantics.html#the-html-element), html is the root element.